src/mappingForAaveWalletManager.ts

```diff
@@ -91,7 +91,17 @@ export function handleWalletDischarged(event: WalletDischarged): void {
 }
 
 export function handleWalletDischargedForCreator(event: WalletDischargedForCreator): void {
-  log.info('TODO: handleWalletDischargedForCreator', []);
+  const aaveSmartWallet = loadOrCreateAaveSmartWallet(event.params.contractAddress, event.params.tokenId);
+  const assetTokenBalance = loadOrCreateAaveAssetTokenBalance(aaveSmartWallet.id, event.params.assetToken, event.params.contractAddress, event.params.tokenId);
+  assetTokenBalance.creatorInterestDischarged = assetTokenBalance.creatorInterestDischarged.plus(event.params.receiverAmount);
+  assetTokenBalance.save();
+
+  var eventData = new Array<string>(5);
```

Comment:
Array size should only be 4 here

Reply:
Updated in current commit.  @robsecord Should be ready for final review